cypress/support/commands.js

```diff
@@ -59,4 +67,4 @@ Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', passwo
     window.localStorage.setItem('user', JSON.stringify(user));
     cy.setCookie('auth', response.body.user.token);
   });
-});
+});
```

Comment:
The `login` command is defined here, but it seems to be duplicating the `register` command logic. Ensure that the `login` command is correctly implemented to perform a login action, rather than registering a new user.